geode-benchmarks/src/main/java/org/apache/geode/benchmark/tests/TestConfigFactory.java

```diff
@@ -17,32 +17,35 @@
 
 import static java.util.concurrent.TimeUnit.MINUTES;
 
+import java.util.Properties;
+
+import com.google.common.base.Strings;
+
 import org.apache.geode.perftest.TestConfig;
 
-public class GeodeBenchmark {
+public class TestConfigFactory {
 
-  /**
-   * Warm up time for the benchmark running on the default runner
-   */
   private static final long WARM_UP_TIME = MINUTES.toSeconds(1);
 
-  /**
-   * Total duration for which the benchmark will run on the default runner
-   */
   private static final long BENCHMARK_DURATION = MINUTES.toSeconds(5);
 
-  /**
-   * Number of threads to run benchmark.
-   */
-  private static final int THREADS = Runtime.getRuntime().availableProcessors() * 16;
+  public static final int DEFAULT_THREADS_PER_PROCESSOR = 16;
```

Comment:
This seems a little confusing given that it may not be the default used. If it doesn't take too much effort, it would be nice for the per-test defaults to be the only defaults that are set, and to have them be set all in the same place